src/main/kotlin/com/sourcegraph/cody/ui/CodyToolWindowFactory.kt

```diff
@@ -419,9 +419,7 @@ class WebUIProxy(private val host: WebUIHost, private val browser: JBCefBrowserB
       val browser =
           JBCefBrowserBuilder()
               .apply {
-                val isOffScreenRenderingSupported =
-                    ApplicationInfo.getInstance().build.baselineVersion >= 232
```

Comment:
What's the background of this change? Isn't it the matter of the app version anymore?